requirements-dev.txt

```diff
@@ -1,3 +1,5 @@
 pre-commit==3.2.1
 pytest-cov>=4.1.0
 docker-squash>=1.1.0
+onnx_graphsurgeon
```

Comment:
The dev requirements don't need to be updated. They're installed on top of the regular requirements.